shared/src/authorization/authorizationClientService.ts

```diff
@@ -21,6 +21,7 @@ export const ROLE_PERMISSIONS = {
   COURT_ISSUED_DOCUMENT: 'COURT_ISSUED_DOCUMENT',
   CREATE_ORDER_DOCKET_ENTRY: 'CREATE_ORDER_DOCKET_ENTRY',
   CREATE_USER: 'CREATE_USER',
+  DISMISS_30_DAY_ALERT: 'DISMISS_30_DAY_ALERT',
```

Comment:
Nit: Could this be made a bit more specific? What is the alert for?

Reply:
Good idea, I renamed it to `DIMISS_NOTT_REMINDER`, does that sound better?